doc/nrf/links.txt

```diff
@@ -548,6 +548,7 @@
 .. _`Memfault`: https://memfault.com/
 .. _`Memfault Docs`: https://docs.memfault.com/
 .. _`Memfault SDK`: https://docs.memfault.com/docs/embedded/introduction/
+.. _`Memfault nRF Connect SDK integration guide`: https://docs.memfault.com/docs/mcu/nrf-connect-sdk-guide#integration-steps
```

Comment:
I think we could drop the `#integration-steps` in the URL to guide users to the top of the page